src/expr/src/relation/func.rs

```diff
@@ -562,24 +626,20 @@ where
         })
         .unzip();
 
-    let result = lag_lead_inner(unwrapped_args, lag_lead_type, ignore_nulls);
+    let result: Vec<Datum<'a>> = lag_lead_inner(unwrapped_args, lag_lead_type, ignore_nulls);
```

Comment:
```suggestion
    let result = lag_lead_inner(unwrapped_args, lag_lead_type, ignore_nulls);
```
(annotation not needed)

Reply:
Removed